crates/turborepo-scm/src/git.rs

```diff
@@ -64,7 +66,38 @@ pub fn changed_files(
     Ok(files)
 }
 
+// Gets the system version of both paths by calling `fs_util::canonicalize`,
```

Comment:
Can you update this comment to be specific about "both paths"? Since there are 3 args to the function, I can't tell at a glance which is being normalized